protocol/0019-MCAL-margin_calculator.md

```diff
@@ -280,6 +280,8 @@ The margin account should have additional amount `limit price x size x margin fa
 
 - For each market and each party which has either orders or positions on the market, the API provides the maintenance margin levels.  (<a name="0019-MCAL-092" href="#0019-MCAL-092">0019-MCAL-092</a>)
 
+- For each market and each party which has either orders or positions on the market, the API provides the current margin mode and, when in isolated margin mode, leverage setting.  (<a name="0019-MCAL-142" href="#0019-MCAL-142">0019-MCAL-142</a>)
```

Comment:
does this ac apply all of these apis?

- ListMarginLevels
- ObserveMarginLevels
- EstimateMargin

Reply:
`EstimateMargin` is going to be a different one, as that is a standalone calculation, but the other two yes

Reply:
The margin ratio, I could make that clearer :D